### PR TITLE
Handle config coming from CONFIG_D directory

### LINKED ISSUE #20: Config cannot be loaded when using CONFIG_D
In the README:

> Add the settings for your PowerDNS API to Dehydrated's config (in /etc/dehydrated or /usr/local/etc/dehydrated)

It only works when putting the hook config inside the "config" file but not if you put it inside a custom config file in the CONFIG_D directory.

BTW thanks for the script :)

### PR DESCRIPTION
**Pull request by bohwaz**
Fix #20 

Using some code from Dehydrated

### CODE REVIEW

**Comment by silkeh** [comment]
Thanks! Can you fix the following `shellcheck` complaints?

```
In pdns_api.sh line 106:
    [[ -n "${ZSH_VERSION:-}" ]] && set +o noglob || set +f
                                ^-- SC2015: Note that A && B || C is not if-then-else. C may run when A is true.

In pdns_api.sh line 119:
    [[ -n "${ZSH_VERSION:-}" ]] && set -o noglob || set -f
                                ^-- SC2015: Note that A && B || C is not if-then-else. C may run when A is true.
```

**Comment by bohwaz** [comment]
Sorry no idea what that line does, just copied the code from dehydrated :)

**Comment by silkeh** [comment]
In that case: 1) add a note that the code is based on dehydrated, and 2) add `# shellcheck disable=SC2015` above the two lines.

**Comment by bohwaz** [comment]
OK actually fixed the code, it's better than ignoring a warning. Note added.

File: pdns_api.sh [hunk 1]
```diff
@@ -97,6 +97,38 @@ load_config() {
     . "${CONFIG}"
   fi
 
+  if [[ -n "${CONFIG_D}" ]]; then
```
Comment (silkeh) [inline pdns_api.sh]: I think this should be `${CONFIG_D-}` to resolve an unbound variable error when `CONFIG_D` is not set.

Comment (bohwaz) [reply pdns_api.sh]: Done :)

**Comment by silkeh** [comment]
Merged. Thanks!

**Comment by bohwaz** [comment]
No prob, thanks to you :)

**Pull request merged**